pkgs/by-name/re/ren-find/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ren-find";
+  version = "0-unstable-2024-01-11";
+
+  src = fetchFromGitHub {
+    owner = "robenkleene";
+    repo = "ren-find";
+    rev = "50c40172e354caffee48932266edd7c7a76a20f";
+    hash = "sha256-zVIt6Xp+Mvym6gySvHIZJt1QgzKVP/wbTGTubWk6kzI=";
+  };
+
+  cargoHash = "sha256-pUy8850v4m9P5OuL15qxmDDQYYyae9HFXRbg3b4f3Lw=";
+
+  meta = with lib; {
```

Comment:
This with lib can be removed.

Reply:
It can, but only by changing or removing the `licenses` and `maintainers` entries below. As you likely know, [I'm on a quest](https://github.com/NixOS/nixpkgs/issues/208242#event-12062086940) against top-level `with`, but I have no quarrel with these `meta` entries. @SuperSandro2000 [says](https://github.com/NixOS/nixpkgs/issues/208242#issuecomment-1381792638):

> The problem is not when with is used in meta where it is very local and only covering a few lines. I am completely against changing those treewide.

... and I agree.